lib/teracy-dev-essential/config/host_manager.rb

```diff
@@ -9,6 +9,12 @@ class HostManager < TeracyDev::Config::Configurator
       def configure_common(settings, config)
         @plugins = settings['vagrant']['plugins'] ||= []
 
+        plugin = @plugins.find { |p| p['name'] == PLUGIN_NAME }
+
+        @plugin_aliases = []
+
+        @plugin_aliases = plugin['options']['aliases'] if plugin
```

Comment:
this config could come from common config or each node config, so we need to check on both.

Reply:
see this config for example: https://github.com/teracyhq-incubator/teracy-dev-entry-k8s/blob/develop/config_default.yaml#L75

Reply:
@hoatle I've updated, pls check again